src/js/common/utils/accessible-menus.js

```diff
@@ -340,10 +361,32 @@ export default function addMenuListeners(menuElement) {
     }
   });
 
-  // TODO: Implement this and get rid of MegaMenu usage
-  // menuElement.addEventListener('click', (e) => {
-  //   // Handle opening menus
-  //   // Handle closing menus when clicked away
-  // });
+  // Close when the screen is resized
+  // Useful for when the menu might be hidden on smaller screens (nav menubar)
+  if (closeOnResize) {
+    window.addEventListener('resize', () => closeAll(menuElement));
```

Comment:
Y'know, I'm not sure about accessibility considerations, actually. I did this just because that's the way it was working before and I wanted to replicate all the former behavior.

I think this functionality was for when the menu is resized to small screen (with the "Menu" button) where the menubar is hidden, then resized back up to where it's shown again. It's unexpected to have the menu stay open.

We can definitely revisit this later to make sure it's still a good idea, but for now, I was just replicating functionality that's already there. :man_shrugging: